bin/protonPack

```diff
@@ -73,14 +77,24 @@ async function main() {
         const port = await findPort(getPort(argv));
         const publicPath = getPublicPath(argv);
 
+        const devIconData = await getDevIcon(logoPath);
+        const devIconPath = getSource('.tmp.logo.png');
+        await fsPromises.writeFile(devIconPath, devIconData);
```

Comment:
Can you move this to the helper ?
Is it possible to use .cache ? so we can use a cache ?